evm_trace/geth.py

```diff
@@ -198,10 +199,17 @@ def _create_node(
             node_kwargs.get("last_create_depth")
             and frame.depth == node_kwargs["last_create_depth"][-1]
         ):
+            # If we get here, we are in the process of completing the attributes from
+            # a CREATE(2) node. The data is located at the first frame with the same depth
+            # after the CREATE(2) opcode was found. This idea is copied from Brownie.
```

Comment:
```suggestion
            # a `CREATE2` node. The data is located at the first frame with the same depth
            # after the `CREATE2` opcode was found. This idea is copied from Brownie.
```

also maybe link the code?

Reply:
I was trying to say that this is for both CREATE and CREATE2 but i guess that was not clear

Reply:
the opcode mnemonic is just `CREATE2`
https://eips.ethereum.org/EIPS/eip-1014

Reply:
i clarifizzled